pkg/credentialprovider/apis/registrycredentials/register.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
This API should probably be in staging/src/k8s.io/kubelet/pkg/apis if we expect external repositories to depend on it.

Reply:
We have an action item before beta to stage the entire credential provider package into it's own repo (including the API) so we left it here for now.